src/components/WarningTooltip.js

```diff
@@ -1,14 +0,0 @@
-import { h } from 'preact';
-import ReactTooltip from 'react-tooltip';
-
-export default function WarningTooltip() {
-  return (
-    <ReactTooltip
-      class="jns-tooltip"
-      type="dark"
-      effect="float"
-      border={true}
```

Comment:
We added a thin white border to the tooltip box to improve visibility when GMail is in dark mode.  It looks like React Tooltip v5 says to use CSS for this effect.

Reply:
Thanks for catching this!